internal/build/hooks.go

```diff
@@ -0,0 +1,60 @@
+package build
+
+import (
+	"io/ioutil"
+	"os"
+	"path"
+	"sigs.k8s.io/yaml"
+
+	"github.com/shipa-corp/ketch/internal/errors"
+)
+
+// Shipa holds the decoded results of a shipa.yaml file
+type Shipa struct {
```

Comment:
there is already a type for this https://github.com/shipa-corp/ketch/blob/main/internal/api/v1beta1/ketch_yaml.go#L17
I would say that we should use it and add `Build []string` to the type.

```bash
ketch app deploy --ketch-yaml=<path-to-ketch-yaml> --strict
```
With --strict parameter set ketch validates `ketch.yaml` and it would be great to validate hooks too.